frontend/src/components/CardContainer.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Card from "react-bootstrap/Card";
+import Container from "react-bootstrap/Container";
+
+const CardContainer = props => {
+
+    const {
+        modals,
```

Comment:
you can put these object instead of props on line 5. const CardContainer = {prop1, prop2, ..} =>